annubes/task.py

```diff
@@ -331,7 +333,7 @@ def plot_trials(self, n_plots: int = 1) -> go.Figure:
                     name="START",
                     mode="markers+lines",
                     x=self.time,
-                    y=self._inputs[i][:, self.n_inputs],  # not for start cue
+                    y=self._inputs[i][:, self.n_inputs - 1],
```

Comment:
What did it mean?

Reply:
That we are doing `-1` (which apparently got lost) is because we don't want a `y` for the start cue, but it is part of `n_inputs`. But if that comment wasn't self explanatory, then it wasn't useful :D